akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/layout/SetScreenshareAsContentReqMsgHdlr.scala

```diff
@@ -0,0 +1,51 @@
+package org.bigbluebutton.core.apps.layout
+
+import org.bigbluebutton.ClientSettings.getConfigPropertyValueByPathAsBooleanOrElse
+import org.bigbluebutton.common2.msgs._
+import org.bigbluebutton.core.apps.{PermissionCheck, RightsManagementTrait}
+import org.bigbluebutton.core.db.LayoutDAO
+import org.bigbluebutton.core.models.Layouts
+import org.bigbluebutton.core.running.OutMsgRouter
+
+trait SetScreenshareAsContentReqMsgHdlr extends RightsManagementTrait {
+  this: LayoutApp2x =>
+
+  val outGW: OutMsgRouter
+
+  def handleSetScreenshareAsContentReqMsg(msg: SetScreenshareAsContentReqMsg): Unit = {
+
+    val allowScreensharePresentationSwitch = getConfigPropertyValueByPathAsBooleanOrElse(
```

Comment:
If you're looking to validate this in akka-apps, it means you want to fully disable the feature rather than just hiding the client controls. I recommend using `disabledFeatures` for this, as that's the approach we've been using to disable features recently.

Reply:
IMO you could simply add a flag to control whether the button is displayed in the layout or not, and leave `disableFeatures` for a future PR if requested.